CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 12.0.0 2017-12-XX
+* `STPRedirectContext` will no longer retain itself for the duration of the redirect, you must explicitly maintain a reference to it yourself. If you 
```

Comment:
incomplete :P

Reply:
Oh weird. I typed more stuff locally into Atom and it seems to not have saved the file before I comitted -_- Will fix